package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "onboard-notify-react",
-  "version": "0.13.3",
+  "version": "0.13.3-0.0.2",
   "dependencies": {
     "bnc-notify": "^1.9.1",
```

Comment:
Can we update to `1.9.2`?

Reply:
I have a separate PR for the version update